git_dumper.py

```diff
@@ -10,14 +10,17 @@
 import sys
 import traceback
 import urllib.parse
+
 import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
We already call this in the main(), so I think that's not necessary. Could you remove it?

Reply:
Sure, I added it there because I was still getting warnings for some reason without it, but I'll remove it.